src/app-logic/url-handling.js

```diff
@@ -594,6 +594,66 @@ export function stateFromLocation(
   };
 }
 
+// Converts "7_10-6-2_5" to [7, 8, 9, 10, 6, 2, 3, 4, 5]
+function convertUintListFromString(s: string): number[] {
```

Comment:
I think these 2 functions could move to `src/utils/uintarray-encoding.js` with better names and possibly some unit tests in `src/test/unit`. That would make their behavior more explicit.

Alternative to the unit tests is adding a few tests to `src/test/url-handling.test.js` (I think we don't have any for the ordering or hiding yet).

Reply:
I did the moving and the renaming and the test adding. Re-requesting review because it was no small change.